### ISSUE TITLE
从标准的yolov5s.pt转换过来的efficient-yolov5s.pt对相同的数据集验证效果有偏差

### ISSUE THREAD

**Issue by Jayus-su** [AlibabaResearch/efficientteacher#99]
作者您好，准确率和mAP.5:.95大概低了两个点，召回率和mAp.5低了零点几，请问这种现象是正常的吗？是因为验证部分的代码机制不一样吗？

**Comment by Shuixin-Li** [comment]
我的是一样的，根据我的经验（1）注意在yolov5 中使用 val.py时候不要设置 --task test，而是 --task val,因为efficientteacher这里的 val.py默认使用的是 validation set 而不是 test set。（2）在写yaml文件的时候要把所有的参数改成你跑完yolo以后的runs/train/exp/opt.yaml里一样的数值

**Comment by jaideep11061982** [comment]
@Shuixin-Li 
<img width="622" alt="image" src="https://github.com/AlibabaResearch/efficientteacher/assets/38164196/85b8260b-f982-47db-8648-62885008d2d9">
Why do we get two results, how to check the performance of the model on Target data which is unlabelled.

2) Which weights to use to predict the unlabelled data labels?

**Comment by lwj1234** [comment]
请问你是怎么把yolov5转成efficient的，你的yaml文件和作者一样吗
